subsys/bluetooth/controller/hci/hci.c

```diff
@@ -5070,8 +5070,9 @@ static void le_ext_adv_report(struct pdu_data *pdu_data,
 	if (!data_status) {
 		uint8_t data_max_len;
 
-		data_max_len = ADV_REPORT_EVT_MAX_LEN - sizeof(*sep) -
-			       sizeof(*adv_info);
+		data_max_len = ADV_REPORT_EVT_MAX_LEN -
+			       sizeof(struct bt_hci_evt_le_meta_event) -
+			       sizeof(*sep) - sizeof(*adv_info);
```

Comment:
Please add my commits from https://github.com/zephyrproject-rtos/zephyr/pull/35451 to this PR, and correct it (that one can be closed with merging, thereafter). So that periodic advertising report too is fixed, if required?